server/index.ts

```diff
@@ -0,0 +1,27 @@
+import express, { Express, Request, Response } from 'express';
+import dbConnect from './lib/dbConnect';
+import { PORT } from './utils/config';
+import icecreamRouter from './controllers/icecreams';
+import userRouter from './controllers/users';
+import orderItemRouter from './controllers/orderitems';
+
+const app: Express = express();
+const port = PORT;
+
+app.use('/api/icecreams', icecreamRouter);
```

Comment:
Looks like this includes work that @pearlescence-m and I did earlier. Would it make sense that I add myself as an assignee on this?

Reply:
@larkinds sure, but since we had a separate branch for our previous work, I created another PR #38 for the previous work. Do you think if it would be okay if we first merge that PR on `backend-initial` branch and this branch only after that?

Reply:
Ahhh gotcha perfect! That works :)